protocol/synthetix/contracts/interfaces/IRewardsManagerModule.sol

```diff
@@ -111,14 +111,16 @@ interface IRewardsManagerModule {
      * @param amount The amount of rewards to be distributed.
      * @param start The date at which the rewards will begin to be claimable.
      * @param duration The period after which all distributed rewards will be claimable.
+     * @return cancelledAmount the amount of reward which was previously issued on a call to `distributeRewards`, but will ultimately not be distributed due to
+     * the duration period being interrupted by the start of this new distribution
      */
     function distributeRewards(
         uint128 poolId,
         address collateralType,
         uint256 amount,
         uint64 start,
         uint32 duration
-    ) external;
+    ) external returns (uint256 cancelledAmount);
```

Comment:
just realised this is a breaking change. Can we have a separate view for cancelled amount instead of updating this function signature please?

Reply:
add a return value actually does not change the signature. try it out!

Reply:
previously no functions used any return value from this function so adding one here wont break previous impls